src/block.tsx

```diff
@@ -52,6 +64,12 @@ export const createRenderChildText = (
                 {element}
               </a>
             );
+          case "p":
+            return (
```

Comment:
I'm thinking of adding some `hooks` here to allow changing page url, what do you think @timolins @tobiaslins?
For example:
```jsx
const pageUrl = hooks?.setPageUrl?.(decorator[1]) ?? decorator[1];
return (
    <a className="notion-link" href={pageUrl} key={i}>
    {element}
    </a>
);
```
This is useful for those want to map `pageId` to `slug` (well, me :smile: ):
```jsx
      hooks={{
        setPageUrl: (pageId) => props.postsIndex?.[pageId] ?? pageId,
      }}
```
Some hooks I could think of:
- **setLinkUrl**: So you could extract block Id to add in page link
- **setPageUrl**: Change page url in case of using `slug`
- **setBlockAttributes** (not implemented): Add `id` attribute for a block so in page link could work, this also allow us to set id for block which has a link in page pointed to only, avoiding adding a bunch of id  which will increasing HTML file size by a lot. Could also use for foot notes.